ArduCopter/mode_guided.cpp

```diff
@@ -355,14 +355,14 @@ void ModeGuided::angle_control_start()
     guided_angle_state.climb_rate_ms = 0.0f;
 }
 
-// set_pos_neu_m - sets guided mode's target pos_neu_m
+// set_pos_ned_m - sets guided mode's target pos_ned_m
 // Returns true if the fence is enabled and guided waypoint is within the fence
 // else return false if the waypoint is outside the fence
-bool ModeGuided::set_pos_NEU_m(const Vector3p& pos_neu_m, bool use_yaw, float yaw_rad, bool use_yaw_rate, float yaw_rate_rads, bool relative_yaw, bool is_terrain_alt)
+bool ModeGuided::set_pos_NED_m(const Vector3p& pos_ned_m, bool use_yaw, float yaw_rad, bool use_yaw_rate, float yaw_rate_rads, bool relative_yaw, bool is_terrain_alt)
 {
 #if AP_FENCE_ENABLED
     // reject destination if outside the fence
-    const Location dest_loc(pos_neu_m * 100.0, is_terrain_alt ? Location::AltFrame::ABOVE_TERRAIN : Location::AltFrame::ABOVE_ORIGIN);
+    const Location dest_loc = Location(is_terrain_alt ? Location::AltFrame::ABOVE_TERRAIN : Location::AltFrame::ABOVE_ORIGIN, pos_ned_m);
```

Comment:
Similar to a comment I've made higher up, I think this line can be changed to

`const Location dest_loc{is_terrain_alt ? Location::AltFrame::ABOVE_TERRAIN : Location::AltFrame::ABOVE_ORIGIN, pos_ned_m};`

Reply:
Thanks for explaining.